starfish/experiment/experiment.py

```diff
@@ -81,6 +79,26 @@ def __repr__(self):
             f"    {images}"
         )
 
+    def parse_coordinate_groups(self, tileset: TileSet) -> List[CropParameters]:
+        coord_groups: Dict[tuple, CropParameters] = dict()
+        for tile in tileset._tiles:
+            x_y_coords = tuple(i for i in [
+                tile.coordinates[Coordinates.X][0], tile.coordinates[Coordinates.X][1],
+                tile.coordinates[Coordinates.Y][0], tile.coordinates[Coordinates.Y][1],
+            ])
+            if x_y_coords in coord_groups:
+                crop_params = coord_groups[x_y_coords]
+                crop_params.add_permitted_axes(Axes.CH, tile.indices[Axes.CH])
+                crop_params.add_permitted_axes(Axes.ROUND, tile.indices[Axes.ROUND])
+                if Axes.ZPLANE in tile.indices:
+                    crop_params.add_permitted_axes(Axes.ZPLANE, tile.indices[Axes.ZPLANE])
```

Comment:
Somewhere in our documentation needs to be explicitly listed that we are ONLY offering registration on `(x, y)` coordinates. @neuromusic should verify this assumption with our users -- that *FISH data does not have misalignment over the z-axis. 

I think this is correct, but we should make sure to communicate it. :-)